packages/homepage/src/screens/home/hero/BoxAnimation/index.js

```diff
@@ -129,6 +150,32 @@ export default function App({ boxes, showPlane }) {
     }, 300);
   }, [boxes, set]);
 
+  if (animationDisabled) {
+    return (
+      <div
+        style={{
+          position: 'absolute',
```

Comment:
please create elements for this instead of adding the css like this :(

Reply:
Changed it! Also changed another one, this whole file is a mess because of how quickly it was built.